test/dataverseClient.spec.ts

```diff
@@ -1481,4 +1483,32 @@ describe('DataverseClient', () => {
       })
     })
   })
-})
\ No newline at end of file
+
+  describe('updateDataset()', () => {
+    it('should call axios with expected url', async () => {
+      const datasetId = random.number().toString()
+      const datasetInformation: BasicDatasetInformation = {
+        title: 's',
```

Comment:
the same random library could be used to generated alphanumerics or words :)

Reply:
Done, I also made use of faker module to generate Names and Dates.